Public/Invoke-DockerTag.ps1

```diff
@@ -1,33 +1,35 @@
 function Invoke-DockerTag {
     [CmdletBinding()]
     param (
+        [ValidateNotNullOrEmpty()]
         [String]
-        $SourceRegistry = '',
+        $Registry,
 
         [Parameter(mandatory=$true)]
         [String]
-        $SourceImage,
+        $ImageName,
 
         [ValidateNotNullOrEmpty()]
         [String]
-        $SourceTag = 'latest',
+        $Tag = 'latest',
 
+        [ValidateNotNullOrEmpty()]
         [String]
-        $TargetRegistry = '',
+        $NewRegistry,
 
         [Parameter(mandatory=$true)]
         [String]
-        $TargetImage,
+        $NewImageName,
 
         [ValidateNotNullOrEmpty()]
         [String]
-        $TargetTag = 'latest'
+        $NewTag = 'latest'
     )
 
```

Comment:
Would it make sense to check if $NewTag -eq $Tag to be able to return immediately if that's the case?

Reply:
What do you mean by return immediately?
Like do nothing when both `Tag` and `NewTag` have the same value?
If that's the case, then the comparison will involve comparing if `$source -eq $target`, where `source/target = ${Registry}${ImageName}${Tag}`.